pkg/kubelet/apis/kubeletconfig/validation/validation_test.go

```diff
@@ -54,7 +54,7 @@ func TestValidateKubeletConfiguration(t *testing.T) {
 
 	errorCase := &kubeletconfig.KubeletConfiguration{
 		CgroupsPerQOS:               false,
-		EnforceNodeAllocatable:      []string{"pods"},
+		EnforceNodeAllocatable:      []string{"pods", "system-reserved", "kube-reserved", "illegal-key"},
```

Comment:
can we add system-reserved and kube-reserved to the success case too?

Reply:
OK, will do :-)

Reply:
Done.